RecoTracker/IterativeTracking/python/iterativeTk_cff.py

```diff
@@ -43,6 +43,25 @@
     InitialStepPreSplitting +
     InitialStep +
     HighPtTripletStep +
+    DetachedQuadStep +
```

Comment:
Ciao Slava,
as Matti said already, it was an important milestone for us, in the process of merging SLHC into the regular release branch, to have the same benchmark that has been used to produce the plots for the technical proposal. It's mainly there for reference, so that we can also target our improvements wrt a very solid starting point (physics wise, omitting performance constrains). On the opportunity to keep it around and alive for a longish period, we can discuss. For sure our focus will be completely on the tuning of the "real" Phase1-Run2-derived tracking.